pyproject.toml

```diff
@@ -0,0 +1,107 @@
+[tool.poetry]
+name = "ayon-core"
+version = "0.3.0"
+description = ""
+authors = ["Ynput Team <team@ynput.io>"]
+readme = "README.md"
+
+[tool.poetry.dependencies]
```

Comment:
Why are these here?

Reply:
to run at least some of the unit tests, but maybe this is for another PR in a future solving development environment with stuff coming from dev bundle. Otherwise just `pre-commit` and `codespell` is needed.